src/vbk/pop_service/pop_service_impl.cpp

```diff
@@ -66,8 +66,8 @@ void clearTemporaryPayloadsImpl(VeriBlock::PopService& pop, uint32_t begin, uint
     for (uint32_t height = begin; height < end; ++height) {
         try {
             pop.removePayloads(heightToHash(height), height);
-        } catch (const std::exception& e) {
-            LogPrintf("removePayloads failed on height %s, but this is okay.", height);
+        } catch (const std::exception&) {
+            LogPrintf("removePayloads failed on height %s, but this is okay. \n", height);
```

Comment:
remove this line pls. Marty says it is useless for users

Reply:
fixed